pkg/services/policer/check.go

```diff
@@ -106,14 +150,23 @@ func (p *Policer) processNodes(ctx *processPlacementContext, addr oid.Address,
 		default:
 		}
 
-		if p.netmapKeys.IsLocalKey(nodes[i].PublicKey()) {
+		if nodes[i].IsMaintenance() {
+			// consider nodes under maintenance (even local) as problem ones.
```

Comment:
why?

Reply:
* see #1812

Reply:
Decided to consider nodes under maintenance as OK in terms of Policer. To speed up reaction to mode switch, status response analysis added.